docs/src/quickstart/build.rst

```diff
@@ -18,6 +18,8 @@ one may want to read more about
 There are several ways to build Cython code:
 
  - Write a setuptools ``setup.py``. This is the normal and recommended way.
+ - Run the ``cythonize`` command-line utility. This is a good approach for
+   compiling a single Cython source file directly to an extension.
```

Comment:
```suggestion
   compiling a single Cython source file directly to an extension.
   A source file can be built "in place" (so that the extension module is generated next to the source file) with
  ``cythonize -i filename.pyx``.
```